airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/CreateConnectorModal.tsx

```diff
@@ -84,13 +80,21 @@ const ErrorText = styled.div`
   max-width: 400px;
 `;
 const validationSchema = yup.object().shape({
-  name: yup.string().required("form.empty.error"),
-  documentationUrl: yup.string().required("form.empty.error"),
-  dockerImageTag: yup.string().required("form.empty.error"),
-  dockerRepository: yup.string().required("form.empty.error"),
+  name: yup.string().trim().required("form.empty.error"),
+  documentationUrl: yup.string().trim().url("form.url.error").notRequired(),
+  dockerImageTag: yup.string().trim().required("form.empty.error"),
+  dockerRepository: yup.string().trim().required("form.empty.error"),
 });
 
-const CreateConnectorModal: React.FC<IProps> = ({ onClose, onSubmit, errorMessage }) => {
+const Label: React.FC<React.PropsWithChildren<unknown>> = ({ children }) => {
+  return (
+    <Text as="span" bold size="lg" className={styles.label}>
```

Comment:
`as="span"` allows us to use the built in `message` prop on the input to display any validation errors